edx_proctoring/rules.py

```diff
@@ -0,0 +1,16 @@
+"Django Rules for edx_proctoring"
+from __future__ import absolute_import
+
+import rules
+
+
+@rules.predicate
+def is_in_reviewer_group(user, attempt):
```

Comment:
Does the documentation for user-based permissions cover testing predicates? Is there a way to do that? I realize that you're indirectly testing the `is_in_reviewer_group` predicate via the tests in `test_reviews.py`, but I wonder whether it's useful to have some unit testing of predicates.

Reply:
I think all the documentation we have on this is https://github.com/edx/edx-platform/blob/9816cadc7168b3d5320b92f68a0097dce7e713e5/lms/djangoapps/courseware/local_decisions/permissions.rst#convert-track-membership-tests-to-permissions

Reply:
There's more documentation in [OEP-9](https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0009-bp-permissions.rst#rule-based-authorization), but it's true that there is a gap in the documentation around permissions, and it's still unclear how best to override permissions in edx-platform in a way that isn't confusing to developers.